session_3/blogspot/app/views/articles/index.html.erb

```diff
@@ -0,0 +1,40 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Articles</h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>Title</th>
+      <th>Topic</th>
+      <th>Tags</th>
+      <th>Content</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @articles.each do |article| %>
+      <tr>
+        <td><%= article.title %></td>
+        <td><%= article.topic %></td>
+        <td><%= article.tags %></td>
+        <td><%= article.content %></td>
+        <% begin %>
+          <% if article.id == current_user.id %>
```

Comment:
Can you use `can? :show, article` and similar helpers instead?

https://github.com/CanCanCommunity/cancancan#check-abilities

Reply:
Yeah, i tried that but in that case, he will see options (edit, show, destroy) for those articles also which he has not created and when he'll click on any of them not associated to him, he'll see unauthorized access error.
So, I'm showing only those article's options which are created by the user.